package/credentials-fetcher.spec

```diff
@@ -4,7 +4,11 @@
  
 # For handling bump release by rpmdev-bumpspec and mass rebuild
 %global baserelease 0
- 
+
+# By default, RPM tries to create debug packages when building RPMs (https://superuser.com/questions/1091529/rpm-build-error-empty-files-file-debugfiles-list)
```

Comment:
This error is not seen in production when building rpms in Amazon Linux. We need to figure out the reason for this error in local rpm builds before making this change